src/Chapter3.hs

```diff
@@ -910,6 +1073,20 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+instance Append [a] where
+    append :: [a] -> [a] -> [a]
+    append x y = x ++ y
+
+newtype Gold = Gold Int
+
+instance Append Gold where
+    append :: Gold -> Gold -> Gold
+    append (Gold x) (Gold y) = Gold (x+y)
```

Comment:
Great! Note that binary operators should be separated with spaces from the arguments 👾 
```suggestion
    append (Gold x) (Gold y) = Gold (x + y)
```